graphql_client/__init__.py

```diff
@@ -45,11 +45,13 @@ def _conn_init(self, headers=None):
         self._conn.send(json.dumps(payload))
         self._conn.recv()
 
+
     def _start(self, payload):
         _id = gen_id()
         frame = {'id': _id, 'type': 'start', 'payload': payload}
         self._conn.send(json.dumps(frame))
-        self._conn.recv()
+        # commenting this as the spec excepts data after start and subscription misses the first data if data is received here
+        # res = self._conn.recv()
```

Comment:
can you remove these commented lines?

Reply:
I have removed the commented lines.

Reply:
https://github.com/ecthiender/py-graphql-client/pull/3/commits/e486ccb616fb3286247dce50f14486a784e7444e